### PR TITLE
Kvg upgrade guppy

### PR DESCRIPTION
**Pull request by kvg**
Update Guppy basecaller to version 6.0.1.

### CODE REVIEW

File: wdl/tasks/Utils.wdl [hunk 1]
```diff
@@ -965,7 +965,7 @@ task MergeBams {
             -p -c --no-PG \
             -@ 2 \
             --write-index \
-            -o "~{prefix}.bam##idx##~{prefix}.bam.bai" \
+            "~{prefix}.bam##idx##~{prefix}.bam.bai"
```
Comment (SHuang-Broad) [inline wdl/tasks/Utils.wdl:968]: why?

File: docker/lr-guppy/Dockerfile [hunk 2]
```diff
@@ -6,4 +6,14 @@ RUN apt-get install -y wget lsb-release
 RUN wget -O- https://mirror.oxfordnanoportal.com/apt/ont-repo.pub | apt-key add -
 RUN echo "deb http://mirror.oxfordnanoportal.com/apt xenial-stable non-free" | tee /etc/apt/sources.list.d/nanoporetech.sources.list
 RUN apt-get update
-RUN apt-get install -yq ont-guppy=5.0.16-1~xenial -V --no-install-recommends
+RUN apt-get install -yq ont-guppy=6.0.1-1~xenial -V --no-install-recommends
+
+RUN apt-get install -yq software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update
+RUN apt-get install -yq python3.6 curl && \
```
Comment (SHuang-Broad) [inline docker/lr-guppy/Dockerfile:14]: does it need to be 3.6? I'd say we use 3.9 when possible.

**Review by SHuang-Broad — COMMENTED** [review]
2 comments

File: docker/lr-guppy/Dockerfile (hunk 2 above)
Comment (kvg) [reply docker/lr-guppy/Dockerfile:14]: I had a hard time getting a newer version of python installed for use with the guppy_duplex tool.  Python 3.6 successfully installed and satisfied guppy_duplex's requirements, so I called it a win :).

File: wdl/tasks/Utils.wdl (hunk 1 above)
Comment (kvg) [reply wdl/tasks/Utils.wdl]: Oops - I think I just selected the wrong commit during the rebase. Fixed now.

File: docker/lr-guppy/Dockerfile (hunk 2 above)
Comment (SHuang-Broad) [reply docker/lr-guppy/Dockerfile:14]: as long as the tool demands, we have no choice....

File: wdl/tasks/Utils.wdl [hunk 3]
```diff
@@ -965,7 +965,7 @@ task MergeBams {
             -p -c --no-PG \
             -@ 2 \
             --write-index \
-            -o "~{prefix}.bam##idx##~{prefix}.bam.bai" \
+            -o "~{prefix}.bam##idx##~{prefix}.bam.bai"
```
Comment (SHuang-Broad) [inline wdl/tasks/Utils.wdl:968]: trailing slash

Comment (kvg) [reply wdl/tasks/Utils.wdl]: Good catch - fixed now!

**Review by SHuang-Broad — APPROVED** [review]
👍

**Pull request merged**